chatminer/nlp.py

```diff
@@ -0,0 +1,16 @@
+from transformers import pipeline
+import pandas as pd
+
+
+def add_sentiment(df: pd.DataFrame, lang: str = "en") -> pd.DataFrame:
+    model_path = (
+        "cardiffnlp/twitter-roberta-base-sentiment-latest"
+        if lang == "en"
+        else "cardiffnlp/twitter-xlm-roberta-base-sentiment"
+    )
+
+    sentiment_pipeline = pipeline("sentiment-analysis", model=model_path)
+    df["sentiment"] = [
+        sentiment["label"] for sentiment in sentiment_pipeline(list(df["message"]))
```

Comment:
can we make this more elegant and potentially improve performance by using pandas' [apply](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.apply.html)?

Reply:
Of course, see if it's the way you want it, now.
Thanks for the tip.